<!DOCTYPE html>

<html>

<head>
<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />
<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Raleway">
<style>
body,h1,h2,h3,h4,h5 {font-family: "Raleway", sans-serif;

                      .header-bar {
      display: flex;
      justify-content: space-between;
      align-items: center;
      background-color: #78B7D0; 
      padding: 10px 20px;
      color: white;
    }

    .header-bar .logo {
      font-size: 1.5em;
      font-weight: bold;
    }

    .header-bar .nav-links a {
      color: white;
      text-decoration: none;
      margin-left: 20px;
      transition: color 0.3s ease;
    }

    .header-bar .nav-links a:hover {
      color: #cce6ff;
    }             
                    
.container {padding: 20px;}
  .text-box { 
    border: 1px solid #ddd;
    padding: 20px;
    background-color: #f9f9f9;
    border-radius: 8px;
    box-shadow: 0 2px 4px rgba(0,0,0,0.1);
     margin-bottom: 20px;
  }
  .text-box p {
    margin: 0;
    line-height: 1.6; 
  }
</style>

  <title>Research</title>


<script src="site_libs/header-attrs-2.7/header-attrs.js"></script>
<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
</head>

  <body>
  
<div class="header-bar">
  <div class="logo">StatsGen</div>
   <div class="nav-links">
  <a href=index.html class="w3-bar-item">Home</a>
  <a href="research.html" class="w3-bar-item">Research</a>
  <a href="publications.html" class="w3-bar-item">Publications</a>
  <a href="outreach.html" class="w3-bar-item">Outreach</a>
  <a href="team.html" class="w3-bar-item">Team</a>
   <a href="Resources.html" class="w3-bar-item">Resources</a>
</div>
  </div>





<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->

<div id="our-lab" class="section level3">
<h3>Our Lab</h3>
<!--<div class="figure">
<img src="images/highland_red-sika_hybrid_KenMcInnes.jpg" alt="photo courtesy of Ken McInnes" width="25%" style=float:right; padding:10px />
<p class="caption">
photo courtesy of Ken McInnes
</p>
</div>
-->
<p>When two closely related species meet, there are a variety of different outcomes. They can compete with each other (competitive exclusion); they can change so that they don't compete (character displacement); and/or they can interbreed (hybridization). For each instance of this 'secondary contact', it's incredibly difficult to <i>predict</i> how likely each of these three options are, even between the same two species. This is really important because secondary contact is increasing with human influences on the world. We're bringing closely related species together, and we don't know what's going to happen. Don't panic. This is a delightfully interesting question to tangle with, which brings together field work, genetics, genomics, theory and statistics to answer.</p>

<p>Much of the StatsGen Lab research centres on the question of predictability in evolutionary processes. Can we predict which species will hybridize, or how many hybrids will be in each population? How does environmental variation affect the outcomes at secondary contact? What can genomic data tell us predicting outcomes? What are our theoretical expectations of the process we think are occuring we can see, and how can we use the resulting patterns to infer processes in wild systems? 
</p>
<p>We're a small but mighty group trying to address these questions that can be loosely sorted into three research themes.
</p>
</div>
<div id="Variable-hybrid-zones" class="section level3">
<h3>Variation Among Replicate Hybrid Zones</h3>
<p>If two species meet in more than once place, what do we expect to happen? Do we expect that the rates of hybridization are always the same? No. Firstly, we expect that the number of hybrids in different hybrid zones can be different for reasons entirely due to chance <a href="https://onlinelibrary.wiley.com/doi/full/10.1111/mec.17359">McFarlane et al. 2024</a>. Secondly, we expect the enviroment to matter to these outcomes, particularly when the secondary contact itself might be caused by human influenced environmental changes. This means that, to understand the underlying processes leading the patterns we see, we need to understand a lot about environmental variation, while thinking hard about differences due to chance. This is delightfully complicated, and a topic many people in the lab are working on, including <a href="https://erynmcfarlane.github.io/team.html#Jenna">Jenna on <i>Fescue</i></a>, <a href="https://erynmcfarlane.github.io/team.html#Sargon">Sargon on <i>Peromyscus</i></a>, <a href="https://erynmcfarlane.github.io/team.html#Amanda">Amanda on <i>Colias</i></a>, and <a href="https://erynmcfarlane.github.io/team.html#Marqus">Marqus using simulations</a>. </p>


</div>
<div id="predicting-evolutionary-outcomes" class="section level3">
<h3>Predicting Evolutionary Outcomes</h3>
<p>We use a lot of statistics to try to understand the natural world. Typically, our first step to answer a question is to collect data, and then do a statistical model determine if those parameters (such as the environment, the weather, the year, the species etc.) can explain the differences in what we're interested (such as the number of hybrid individuals in a population). If we see a statistical relationship between these things, awesome, and we then use that model to describe the relationship. Great, except there are many pitfalls to these models, including overconfidence (both statistical and emotional). </p>

Both <a href="https://erynmcfarlane.github.io/team.html#Amandeep">Amandeep</a> and <a href="https://erynmcfarlane.github.io/team.html#Eryn">Eryn</a> have been working on this lately. Amandeep asked how well a genome wide association study (GWAS) done on <a href="https://academic.oup.com/g3journal/article/11/10/jkab274/6335679?login=false">hybrid deer in Scotland </a> (a project Eryn did as a postdoc with <a href="https://biology.ed.ac.uk/josephine-pemberton-group">Josephine Pemberton</a>) could be used to predict the phenotypes of deer that weren't used to build the model. Eryn, as a result of her previous postdoc work with the <a href="https://microcollaborative.atlassian.net/wiki/spaces/MP/overview">modelscape project</a> with <a href="https://cbuerkle.bitbucket.io/">Alex Buerkle</a> is having a blast using simulations to ask how well we can do prediction in general (<a href="https://www.biorxiv.org/content/10.1101/2024.03.15.585297v3.abstract">Jahner et al. in review</a>). Watch this space, this is a problem that the lab is generally thinking quite hard about.
</div>

<div id="methods" class="section level3">
<h3>Statistical Genetics Methods (respectfully, are we doing it wrong?)</h3>
<p> Finally, as in any quantiative lab, we're happily testing new methods on old problems whenever we can. This can mean lots of things, including using simulations so that we can see the patterns that occur when we know the underlying processes, and sometimes, even suggest new methods (oldie but a goodie - <a href="https://besjournals.onlinelibrary.wiley.com/doi/full/10.1111/2041-210X.12535">RepeatABEL lives on</a>, see <a href="https://onunicornsandgenes.blog/2019/11/24/using-r-installing-genabel-and-repeatabel/">Martin Johnsson's excellent description</a> of how to install RepeatABEL given GenABEL has gone offline.). For example, <a href="https://erynmcfarlane.github.io/team.html#Santiago">Santiago</a> has been using publically available genotyping-by-sequencing data to ask whether different GBS databases can be used together, especially for common animals. <a href="https://erynmcfarlane.github.io/team.html#Eryn">Eryn</a> and friend-of-the-lab <a href="https://nmu.edu/biology/liz-mandeville">Liz Mandeville</a> have discussed how to use hierarchical (mixed effects) models to address stocastic and environmental sources of variation in replicate hybrid zones <a href="https://onlinelibrary.wiley.com/doi/full/10.1111/gcb.16527">paper here</a>. This is a point of constant interest for the lab. We know that all models are wrong, but some are useful (thanks George Box!), and some are more useful than others, so we're always keen to look at the tools (statistical and otherwise) to ask if we could do better.
  </p>
<p>Thank you to <a href="https://erynmcfarlane.github.io/team.html#Yosuf">Yosuf</a> who wrote the first draft of this research page! </p>  

</div>


<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
